File: Backend/router/settingRouter.js
const express = require('express');
const settingRouter = express.Router();
const {
  getSettings,
  updateSettings
} = require('../Controllers/Settingcontroller');

/**
 * @route   GET /api/settings
 * @desc    Get application settings
 * @access  Public
 */
settingRouter.get('/api/settings', getSettings);

/**
 * @route   PUT /api/settings
 * @desc    Update application settings
 * @access  Public
 */
settingRouter.put('/api/settings', updateSettings);

module.exports = settingRouter;

File: python-backend/routes/setting_router.py
from flask import Blueprint
from controllers.setting_controller import get_settings, update_settings

# Create blueprint
setting_bp = Blueprint('setting', __name__)

@setting_bp.route('/', methods=['GET'])
def get():
    """
    GET /api/settings
    Get application settings
    """
    return get_settings()

@setting_bp.route('/', methods=['PUT'])
def update():
    """
    PUT /api/settings
    Update application settings
    """
    return update_settings()